Classes/Utils/Messages.cs

```diff
@@ -427,8 +427,23 @@ public class WebMessage {
                         };
 
                         Process.Start(processInfo);
+                        Process.GetCurrentProcess().Kill(); // this is not a clean exit, need to look into why we can't cleanly exit
+
+#else
+                        string applicationPath = Environment.GetCommandLineArgs()[0];
+                        string escapedApplicationPath = applicationPath.Replace("\"", "\\\"");
```

Comment:
Linux support should be removed. Linux application updates may be handled differently in the future so there is no guarantee that we will be needing this.

Reply:
Aight, I'll fix it tomorrow. I just turned off the PC 😀